apps/keyboard/js/imes/jskanji/jskanji.js

```diff
@@ -1139,6 +1137,7 @@
   };
 
   // for non-Mozilla browsers
+  /* jshint ignore:start */
```

Comment:
Actually, I don't this segment of code could do what it intend to, since this has been wrapped in a function block.
Maybe we could add a comment here for that.